integration/ts-compat/helpers.js

```diff
@@ -2,7 +2,7 @@ const {relative, sep, join} = require('path');
 const {readdirSync, readFileSync, existsSync, unlinkSync} = require('fs');
 const {set, ln, rm, mkdir} = require('shelljs');
 const {fork} = require('child_process');
-const runfiles = require(process.env.BAZEL_NODE_RUNFILES_HELPER);
+const {runfiles} = require('@bazeL/runfiles');
```

Comment:
```suggestion
const {runfiles} = require('@bazel/runfiles');
```

Looks like `L` is accidentally capitalized here?

Reply:
yup. me trying to type fast with the laptop 😄